### PR TITLE
Split up the transit editor and fix auto create stops

### PR DESCRIPTION
**Pull request by trevorgerhardt**

### CODE REVIEW

**Comment by mattwigway** [comment]
@trevorgerhardt I'm still seeing the same behavior with this fix (when I draw a new line, stops are automatically created even though the checkbox defaults to off).

**Comment by trevorgerhardt** [comment]
Fixed! Defaults weren't set correctly 👌

**Comment by trevorgerhardt** [comment]
@mattwigway can you check this one out again? Should be good to go

**Comment by mattwigway** [comment]
I'll take a look this afternoon.

File: lib/components/modification/add-trip-pattern.js [hunk 1]
```diff
@@ -77,34 +79,38 @@ export default class AddTripPattern extends DeepEqual {
...
     }
   }
 
   render () {
     const {modification, mapState, numberOfStops, segmentDistances, update} = this.props
-    const {extendFromEnd, followRoad} = this.state
+    const {createStopsAutomatically, extendFromEnd, followRoad, spacing} = this.state
     // not using state here as otherwise checkbox would toggle on and off as you're editing the stop spacing
-    const autoCreateStops = modification.segments.length > 0 ? modification.segments[0].spacing > 0 : true
+    // const autoCreateStops = modification.segments.length > 0 ? modification.segments[0].spacing > 0 : true
```
Comment (trevorgerhardt) [inline lib/components/modification/add-trip-pattern.js]: Yup!

**Pull request merged**